www/src/components/landingPage/community/community.astro

```diff
@@ -7,80 +9,68 @@ export interface Props {}
           ></path>
         </svg>
-        <h2 class="text-xl">Discord</h2>
-        <p class="text-left">
-          Join our Discord server to chat with other developers and get help
-          with your projects.
-        </p>
-      </a>
-      <a
+      </CommunityCard>
+      <CommunityCard
+        title="Github"
+        text="Contribute to the project by submitting issues and pull requests on our Github repository."
         href="https://github.com/t3-oss/create-t3-app"
-        target="_blank"
-        rel="noopener noreferrer"
-        class="flex flex-col items-start justify-center border border-t3-purple-100 rounded-lg p-3 space-y-3 h-48 sm:h-56 hover:bg-white/10 cursor-pointer w-full"
       >
         <svg
           role="img"
           viewBox="0 0 24 24"
           xmlns="http://www.w3.org/2000/svg"
           class="fill-white"
+          slot="icon"
         >
           <title>GitHub</title>
           <path
             d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"
           ></path>
         </svg>
-        <h2 class="text-xl">Github</h2>
-        <p class="text-left">
-          Contribute to the project by submitting issues and pull requests on
-          our Github repository.
-        </p>
-      </a>
-      <a
+      </CommunityCard>
+      <CommunityCard
+        title="Twitter"
+        text="Join us on Twitter for shitposts"
```

Comment:
Not a problem for now, but just noting it down.

We should either fix the styles here or just add some more text so it aligns with the others 
<img width="1303" alt="CleanShot 2022-08-31 at 23 27 00@2x" src="https://user-images.githubusercontent.com/51714798/187788458-392dea5a-2658-40f7-a907-c969bad35cee.png">

Reply:
I'll try to update this tomorrow ;)